lib/src/robot/client.dart

```diff
@@ -38,6 +38,12 @@ class RobotClientOptions {
   /// Convenience initializer for creating options with specified [DialOptions]
   RobotClientOptions.withDialOptions(this.dialOptions);
 
+  // Convenience initializer for creating options with an API key ID/API key pair
+  RobotClientOptions.withAPIKey(String apiKeyID, String apiKey)
```

Comment:
this should be `withApiKey` ([Dart style guide](https://dart.dev/effective-dart/style#do-capitalize-acronyms-and-abbreviations-longer-than-two-letters-like-words) for initialisms longer than 2 letters)

Reply:
oh thank you so much for catching this!